packages/manager/modules/config/package.json

```diff
@@ -13,14 +13,17 @@
     "directory": "packages/manager/modules/config"
   },
   "license": "BSD-3-Clause",
-  "main": "./dist/esm/index.js",
+  "sideEffects": false,
+  "main": "dist/index.js",
+  "module": "dist/index.js",
```

Comment:
Need to define both ?

Reply:
@marie-j I put them both because of what i understood, module is for packages that can handle ES modules. If a package that can use ES Module see the module entry, it will use it instead of main for better use of the package. But for packages who don't understand ES Modules or are written in CommonJS etc, the need the main entry to be able to stuff like `require('myModule')`. 

To be honest, i don't know if i need to define them both, i just thought of main as a fallback if the package couldn't understand the module entry. 

The first answer in this SO topic actually gives a lot of info about it : https://stackoverflow.com/questions/42708484/what-is-the-module-package-json-field-for
But i couldn't find anywhere if i should use it alone or couple it with main too.